pkg/app/piped/executor/kubernetes/kubernetes.go

```diff
@@ -189,7 +189,7 @@ func (e *Executor) addBuiltinAnnontations(manifests []provider.Manifest, variant
 }
 
 func (e *Executor) applyManifests(ctx context.Context, manifests []provider.Manifest) error {
-	e.LogPersister.Infof("Start applying %d manifests", len(manifests))
+	e.LogPersister.Infof("Start applying %d manifests to %q namespace", len(manifests), e.config.Input.Namespace)
```

Comment:
What will be logged when the `Input.Namespace` was not specified?

Reply:
You're quite right. Can't really say which namespace this applies to, so make sure to show when only specified.